android/app/build.gradle

```diff
@@ -139,13 +139,15 @@ android {
     compileSdkVersion rootProject.ext.compileSdkVersion
     buildToolsVersion rootProject.ext.buildToolsVersion
 
+    flavorDimensions "default"
+
     defaultConfig {
-        applicationId "fr.coopcycle"
+        applicationId "fr.naofood.client"
```

Comment:
From what I understand [here](https://stackoverflow.com/questions/48539060/can-i-give-dynamic-placeholder-for-manifest-file-package), it shouldn't be necessary to rename all the files, just changing the `applicationId` is sufficient. This needs to be tested.

Reply:
More on this subject [here](https://stackoverflow.com/a/60001600) and luggit/react-native-config#148

It seems possible to just change `applicationId`, but it needs a small change when launching React Native, because by default it reads `AndroidManifest.xml`

```
$ npx react-native run-android --help

--appId <string>          Specify an applicationId to launch after build. If not specified, `package` from AndroidManifest.xml will be used. (default: "")
```